ironfish/src/mining/miner.ts

```diff
@@ -179,7 +179,7 @@ async function miner(
 
       tasks[randomness] = enqueue(
         piscina,
-        Buffer.from(blockToMineResult.value.bytes),
+        Buffer.from(blockToMineResult.value.bytes.data),
```

Comment:
This is fine, but I'm guessing this logged a type error somewhere? We should probably simplify `newBlockData` at some point

Reply:
Yeah, the build was failing as we upgraded some packages due to types

Reply:
What type does this match? I don't see a Buffer.from constuctor that accepts this? Is this matching the interface for one of the array types? Like ArrayBuffer or UInt8Array?

Reply:
How was this ever working before? What constructor made this work before? Did bytes match the interface for one of the array types?

Reply:
It looks like the old typescript types were wrong, even though buffer does take this as an input.